openreview/venue/invitation.py

```diff
@@ -1357,6 +1357,16 @@ def set_withdrawal_invitation(self):
             },
             'venueid': {
                 'value': self.venue.get_withdrawn_submission_venue_id()
+            },
+            '_bibtex': {
```

Comment:
I don't think you need to define the bibtex field, or are you getting an error?

Reply:
yes, I was getting a `ValidationError` I believe, so I checked the journal and saw it was defined there in some places. I can double check and see exactly what error I was getting.

Reply:
<img width="1490" alt="Screen Shot 2023-06-27 at 10 26 30 AM" src="https://github.com/openreview/openreview-py/assets/32438984/81b28e61-a8bd-462f-8bbb-00fa836d8ee0">